### ISSUE TITLE
Add general information to the header from CoinCap.io

### ISSUE THREAD

**Issue by kazazor** [kazazor/coinsmarket#42]
* They have an api to show cool global information. We should show it as well.
* I wrote in the header, but could be elsewhere..

**Comment by kazazor** [comment]
Is blocked by: https://github.com/kazazor/coinsmarket/issues/52

**Comment by kazazor** [comment]
Won't do that. We can close it.
